storage-bigtable/src/bigtable.rs

```diff
@@ -135,19 +140,33 @@ impl BigTableConnection {
                     instance_name
                 );
 
+                let endpoint = {
+                    let endpoint =
+                        tonic::transport::Channel::from_static("https://bigtable.googleapis.com")
+                            .tls_config(
+                            ClientTlsConfig::new()
+                                .ca_certificate(
+                                    root_ca_certificate::load().map_err(Error::CertificateError)?,
+                                )
+                                .domain_name("bigtable.googleapis.com"),
+                        )?;
+
+                    if let Some(timeout) = timeout {
+                        if read_only {
+                            // apply timeout only when thre is no write queries!
+                            endpoint.timeout(timeout)
+                        } else {
+                            warn!("BigTableConnection's requested timeout configuration is ignored because it's not read-only.");
```

Comment:
I think we can also apply the timeout when read-write and configure `--rpc-bigtable-timeout` higher on nodes with read-write access if necessary (I suspect it won't be necessary though)

Reply:
kk, it's assuring for me to hear that from you. I'm lightly worried how long it takes to write to bigtables in general and general retry handling in the write path. I'm new to this area of code. ;)